src/actions/user_actions.js

```diff
@@ -32,6 +38,14 @@ const connect = async () =>
     safeApp = await window.safe.initialiseApp( appInfo );
     await unregisteredConn();
     console.log( 'Read-only connection created...' );
+
+    try {
+      await safeApp.fetch();
```

Comment:
You can do: `if( ! safeExperimentsEnabled )`

https://github.com/maidsafe/safe_browser/blob/master/app/components/Tab/Tab.jsx#L526

Reply:
Ah cool, I wasn't aware of that and was thinking we needed something like it, I'll try to use that